src/librbd/cache/pwl/ssd/WriteLog.cc

```diff
@@ -166,6 +167,31 @@ bool WriteLog<I>::initialize_pool(Context *on_finish,
       on_finish->complete(r);
       return false;
     }
+
+    bufferlist bl;
+    ::IOContext ioctx(cct, nullptr);
+    bdev->read(0, MIN_WRITE_ALLOC_SSD_SIZE, &bl, &ioctx, false);
```

Comment:
I know you just moved this code, but need to check for errors here:
```suggestion
    r = bdev->read(0, MIN_WRITE_ALLOC_SSD_SIZE, &bl, &ioctx, false);
    if (r < 0) {
      ...
    }
```